tests/build/build_templates.go

```diff
@@ -575,13 +575,19 @@ var _ = framework.BuildSuiteDescribe("Build templates E2E test", Label("build",
 
 						// Refresh our copy of the PipelineRun for latest results
 						pr, err = kubeadminClient.TektonController.GetPipelineRun(pr.Name, pr.Namespace)
+						if err != nil {
+							GinkgoWriter.Printf("The PipelineRun %s in %s namespace %s failed, error message: %s\n", pr.Name, pr.Namespace, err)
```

Comment:
I don't think this is doing what you think it's doing.

My understanding is that `err` represents the error from fetching the PipelineRun from the cluster, e.g. 404 not found. This will not include the status of the PipelineRun. It also doesn't check the status of the PipelineRun. If retrieval of the PipelineRun is successful, but the PipeilneRun itself failed, `err` will be `nil`.

Reply:
@lcarva thank you for the explanation. I'm reworking.

Reply:
@lcarva Can you help to have  another review? Thanks!